src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,81 @@
+package duke;
+
+import duke.command.Command;
+import duke.command.DukeCommandType;
+import duke.task.DukeTaskManager;
+import duke.ui.DukeUi;
+
+/**
+ * Duke is a Personal Assistant Chatbot that helps users to organise their tasks using a command line interface
+ */
+public class Duke {
+
+    private static DukeTaskManager taskManager;
+    private static DukeUi dukeUi;
+
+    /**
+     * Starts the process of running Duke
+     */
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        startService();
+    }
+
+    private static void startService() {
+        dukeUi = new DukeUi();
+        dukeUi.dukePrint("Hello! I'm Duke \nWhat can I do for you?\n");
+        taskManager = new DukeTaskManager(Storage.loadData());
+        run();
+    }
+
+    private static void run() {
+        while (dukeUi.hasNextLine()) {
+            Command command = Parser.getCommand(dukeUi.getNextLine());
+            DukeCommandType commandType = command.getCommandType();
+            String args = command.getArgs();
+            switch (commandType) {
+            case EXIT: {
```

Comment:
Might be personal preference but the braces on switch statements are optional and so you could technically remove them if you want to! Same goes for the other switch statements as well.